README.md

```diff
@@ -1,5 +1,46 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # React Bookstore App
 
-The real README file is in the feature branch
+Create the basic components to the bookstore
+## Built With
 
-Enviroment Setup
\ No newline at end of file
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Javascript
+- React.js
+
+### Install
+Type these commands in your terminal or command line
+- `git clone https://github.com/Dangelo-JAN/react-bookstore-app.git`
+- `npm install`
+
```

Comment:
- [ ] I would recommend including all the steps to clone and run the project.

```
$ git clone https://github.com/Dangelo-JAN/react-bookstore-app.git
$ cd react-bookstore-app
$ npm i
$ npm start
```

Reply:
This is not a recommendation, it was a requirement